src/docquery/ext/pipeline_document_classification.py

```diff
@@ -143,6 +143,7 @@ def preprocess(
         word_boxes: Tuple[str, List[float]] = None,
         lang=None,
         tesseract_config="",
+        max_num_spans=1,
```

Comment:
You should add it to `_sanitize_parameters` too, so that someone can use it while instantiating the pipeline (and then it'll automatically be forwarded to `preprocess`)

Reply:
Done :+1:.